tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -325,14 +324,10 @@ def test_multiple_fsx(
     num_existing_fsx_open_zfs_volumes = (
         2 if partition in ["aws"] else 0
     )  # China, GovCloud and Isolated do not have OpenZFS.
-    if request.config.getoption("benchmarks") and os == "alinux2":
-        # Only create more FSx when benchmarks are specified. Limiting OS to reduce cost of too many file systems
-        num_existing_fsx = 20
-    else:
-        # Minimal total existing FSx is the number of Ontap and OpenZFS plus one existing FSx Lustre
-        num_existing_fsx = num_existing_fsx_ontap_volumes + num_existing_fsx_open_zfs_volumes + 1
+    # Minimal total existing FSx is the number of Ontap and OpenZFS plus one existing FSx Lustre
+    num_existing_fsx = num_existing_fsx_ontap_volumes + num_existing_fsx_open_zfs_volumes + 1
     num_existing_fsx_lustre = num_existing_fsx - num_existing_fsx_ontap_volumes - num_existing_fsx_open_zfs_volumes
-    fsx_lustre_mount_dirs = ["/shared"]  # OSU benchmark relies on /shared directory
+    fsx_lustre_mount_dirs = ["/shared"]
```

Comment:
Same as above: According to the removed comment it seems that the /hsared folder was required only by OSU benchmarks. Is it still required if we remove them?

Reply:
Clarified in a similar comment. Better to keep this in this PR.